<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mainwindow</class>
 <widget class="QWidget" name="mainwindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>321</width>
    <height>272</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>500</width>
    <height>500</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Alarm &amp; Clock </string>
  </property>
  <property name="autoFillBackground">
   <bool>false</bool>
  </property>
  <property name="styleSheet">
   <string notr="true">border-top-color: rgb(170, 255, 127);
background-color: rgb(85, 170, 127);</string>
  </property>
  <widget class="QTabWidget" name="tw">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>308</width>
     <height>261</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(85, 170, 127);</string>
   </property>
   <property name="currentIndex">
    <number>1</number>
   </property>
   <widget class="QWidget" name="Timer">
    <attribute name="title">
     <string>Timer</string>
    </attribute>
    <widget class="QSpinBox" name="sp_hour">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>70</y>
       <width>39</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>100</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hour</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_min">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>70</y>
       <width>39</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>100</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Minute</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="sp_sec">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>70</y>
       <width>39</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>100</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Second</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_timer_start">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>140</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Start</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_timer_pause">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>140</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Pause</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_timer_reset">
     <property name="geometry">
      <rect>
       <x>200</x>
       <y>140</y>
       <width>80</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <pointsize>12</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Reset</string>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_timer">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>151</width>
       <height>31</height>
      </rect>
     </property>
     <property name="font">
      <font>
       <family>Comic Sans MS</family>
       <pointsize>12</pointsize>
       <weight>50</weight>
       <bold>false</bold>
      </font>
     </property>
     <property name="text">
      <string>00:00:00</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="Stopwatch">
    <attribute name="title">
     <string>StopWatch</string>
    </attribute>
    <layout class="QGridLayout" name="gridLayout">
     <item row="1" column="0">
      <widget class="QPushButton" name="btn_stopwatch_start">
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>13</pointsize>
         <weight>50</weight>
         <bold>false</bold>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #333;
background-color: rgb(0, 255, 127);
border: 2px solid #555;
border-radius: 30px;
border-style: outset;

padding: 5px;</string>
       </property>
       <property name="text">
        <string>Start</string>
       </property>
       <property name="iconSize">
        <size>
         <width>30</width>
         <height>30</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Return</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QPushButton" name="btn_stopwatch_pause">
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #333;
background-color: rgb(255, 255, 112);
border: 2px solid #555;
border-radius: 30px;
border-style: outset;

padding: 5px;</string>
       </property>
       <property name="text">
        <string>Pause</string>
       </property>
       <property name="shortcut">
        <string>Space</string>
       </property>
      </widget>
     </item>
     <item row="1" column="2">
      <widget class="QPushButton" name="btn_stopwatch_reset">
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>13</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: #333;
background-color: rgb(255, 123, 106);
border: 2px solid #555;
border-radius: 30px;
border-style: outset;

padding: 5px;</string>
       </property>
       <property name="text">
        <string>Reset</string>
       </property>
       <property name="shortcut">
        <string>Del</string>
       </property>
      </widget>
     </item>
     <item row="1" column="3">
      <widget class="QPushButton" name="btn_record">
       <property name="font">
        <font>
         <family>Kristen ITC</family>
         <pointsize>16</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">background-color: rgb(170, 255, 255);</string>
       </property>
       <property name="text">
        <string>Record</string>
       </property>
       <property name="shortcut">
        <string>R</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="QTableWidget" name="tbl_records">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="verticalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="horizontalScrollBarPolicy">
        <enum>Qt::ScrollBarAlwaysOff</enum>
       </property>
       <property name="sizeAdjustPolicy">
        <enum>QAbstractScrollArea::AdjustToContents</enum>
       </property>
       <property name="textElideMode">
        <enum>Qt::ElideMiddle</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="rowCount">
        <number>0</number>
       </property>
       <property name="columnCount">
        <number>0</number>
       </property>
       <attribute name="horizontalHeaderHighlightSections">
        <bool>false</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item row="0" column="0" colspan="4">
      <widget class="QLabel" name="lbl_stopwatch">
       <property name="font">
        <font>
         <family>MS Serif</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="text">
        <string>00:00:00</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="Alarm">
    <attribute name="title">
     <string>Alarm</string>
    </attribute>
    <widget class="QPushButton" name="btn_active">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>110</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Activate</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_deactive">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>140</y>
       <width>80</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Deactivate</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="alarm_hour">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>50</y>
       <width>39</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Second</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QSpinBox" name="alarm_min">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>50</y>
       <width>39</width>
       <height>31</height>
      </rect>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="alarm_sec">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>50</y>
       <width>39</width>
       <height>31</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>80</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Minute</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>80</y>
       <width>31</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Hour</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QLabel" name="lbl_alarm">
     <property name="geometry">
      <rect>
       <x>156</x>
       <y>20</y>
       <width>81</width>
       <height>20</height>
      </rect>
     </property>
     <property name="layoutDirection">
      <enum>Qt::LeftToRight</enum>
     </property>
     <property name="text">
      <string>OFF</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
    <widget class="QWidget" name="">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>70</width>
       <height>136</height>
      </rect>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QCheckBox" name="chb_sat">
        <property name="text">
         <string>Sat</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chb_sun">
        <property name="text">
         <string>Sun</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chb_mon">
        <property name="text">
         <string>Mon</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chb_thu">
        <property name="text">
         <string>Thu</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chb_wen">
        <property name="text">
         <string>Wen</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chb_thr">
        <property name="text">
         <string>Thr</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chb_fri">
        <property name="text">
         <string>Fri</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
